deploy/rosa-oauth-templates/config.yaml

```diff
@@ -3,4 +3,6 @@ selectorSyncSet:
   matchExpressions:
   - key: api.openshift.com/product
     operator: In
-    values: ["rosa"]
\ No newline at end of file
+    values: ["rosa"]
+  applyBehavior: "CreateOrUpdate"
+  resourceApplyMode: "Upsert"
```

Comment:
```suggestion
  # use Upsert so if this configuration no longer applies it will not delete the OAuth resource in cluster which is a singleton.  We should never delete OAuth.
  resourceApplyMode: "Upsert"
```